Snowflake.Data/Core/FileTransfer/StorageClient/SFSnowflakeAzureClient.cs

```diff
@@ -54,6 +54,12 @@ public SFSnowflakeAzureClient(PutGetStageInfo stageInfo)
             }
         }
 
+        internal SFSnowflakeAzureClient(PutGetStageInfo stageInfo, BlobServiceClient mockClient) : this(stageInfo)
+        {
+            // Inject the mock BlobServiceClient
+            blobServiceClient = mockClient;
```

Comment:
can we rename mockClient to mockBlobClient or mockBlobService to be more self explanatory

Reply:
Sure